/**:
  ros__parameters:
    frequency: 100
    init_pose: [-63.0, -99.0, 3.1415] # x, y, psi
    parent_frame: "odom"
    child_frame: "base_link"

# PID Controller for the VTec SDV
vel_pid_node:
  ros__parameters:
    Kp: 0.5
    Ki: 0.1
    Kd: 0.1
    # Sims solo en rviz:
    # Kp: 1.0
    # Ki: 0.5
    # Kd: 0.2
    U_MAX: 1.0
    U_MIN: -1.0
    enable_ramp: true
    ramp_rate: 0.1

# Stanley Controller Params
stanley_controller_node:
  ros__parameters:
    K: 1.0 # drives you to path faster, induces overshoot
    K_soft: 5.0 # softens overshoot
    DELTA_SAT: [-0.5264, 0.5922] # new car limits in radians
